ping/index.test.js

```diff
@@ -98,6 +98,33 @@ it('sends only one ping if timeout is bigger than ping', async () => {
   expect(test.leftSent).toEqual([['ping', 1]])
 })
 
+it('do not try clear timeout if it does not set', async () => {
+  let test = await createTest({
+    ping: 100,
+    timeout: 300,
+    fixTime: false
+  })
+  Object.defineProperty(test.leftNode, 'pingTimeout',
+    {
+      get: () => undefined,
+      set: timeout => {
+        clearTimeout(timeout)
+      }
+    }
+  )
+  await delay(250)
+
+  expect(test.leftSent).toEqual([['ping', 1]])
+})
+
+it('not send ping if not connected', async () => {
```

Comment:
You need to fix this title as well to make sentence in English

Reply:
Sorry. Fixed it too. Thanks